application/config/config.go

```diff
@@ -146,6 +146,10 @@ type Config struct {
 	integrationVersion          string
 	automaticAuthentication     bool
 	tokenChangeChannels         []chan string
+	filterCriticalSeverity      concurrency.AtomicBool
```

Comment:
can we have a complex object in settings for this? That'd nicely group some of the settings :)

Reply:
done